src/components/ModelDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 -2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+import type { ReactNode } from 'react'
+
+import { ModalContent } from './components/ModalContent'
+import { ModalHeader } from './components/ModalHeader'
+import { StyledModal } from './styled'
+
+export type ModalDialogProps = {
+  onClose: () => void
+  title: string
+  onConfirm?: () => void
+  children?: ReactNode
+  isConfirmation?: boolean
```

Comment:
I would make it mandatory with a default value of `false`

Reply:
Done

Reply:
Done.